subprojects/gradle/ci-logger/src/main/kotlin/com/avito/logger/formatter/AppendMetadataFormatter.kt

```diff
@@ -0,0 +1,29 @@
+package com.avito.logger.formatter
+
+import com.avito.logger.LoggerMetadata
+import com.avito.logger.LoggingFormatter
+
+internal class AppendMetadataFormatter(private val metadata: LoggerMetadata) : LoggingFormatter {
```

Comment:
Expected to see unit test for that class

Reply:
done